### DISCUSSION TITLE
How can we replace the email with phone number

### DISCUSSION THREAD

**Discussion by anisryad** [PrestaShop/PrestaShop#35585] [Q&A]
Good morning,
We would like to be able to replace the email field with a phone number field because most of our future customers may not want to fill in this field if they are in email mode, however all targeted users are used to using their phone number to place orders, this is why we would like to replace this field without modifying the internal functioning of the application by automatically adding on the fly @ at the end of the telephone number entered such as for example: 0123456789 becomes 0123456789@something.xyz before posting the form.

Or another possibility is to make the email field optional and add another field phone number which will be mandatory.

Thank you so much for your help.

**Comment by MatShir** [comment]
Your question is very similar to[ your previous post](https://github.com/PrestaShop/PrestaShop/discussions/35581). I am closing it and suggest commenting on the other post again to avoid duplicate content.

**Reply by anisryad** [reply]
OK i will do this instead
thank you
